components/user/EntryTypeGraph.jsx

```diff
@@ -40,6 +40,34 @@ const EntryTypeGraph = ({ data }) => {
     })
   }
 
+  const playtimePieChartData = []
+  playtimePieChartData.push({
+    name: "OW",
+    percOfTotal: data.ot
+  })
+  playtimePieChartData.push({
+    name: "Nether",
+    percOfTotal: data.nt
+  })
+  playtimePieChartData.push({
+    name: "Wall",
+    percOfTotal: data.wt
+  })
+
+  const netherDistData = []
```

Comment:
This should be done on the backend, its rather intensive

Reply:
will have you change that or I will change later